source/GstreamerCatLog.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024 Sky UK
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation;
+ * version 2.1 of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+#ifndef GSTREAMER_CAT_LOG_H
+#define GSTREAMER_CAT_LOG_H
+
+#include <gst/gst.h>
+
+GST_DEBUG_CATEGORY_EXTERN(rialtoGStreamerCat);
+
+#define INIT_GST_DEBUG_CATEGORY()                                                                                      \
```

Comment:
Thinking about it this should be inline rather than a macro, we should avoid macros if we can according to our guidelines https://google.github.io/styleguide/cppguide.html